docs/en/connector-v2/sink/Phoenix.md

```diff
@@ -0,0 +1,41 @@
+# Phoenix
+
+> Phoenix sink connector
+
+## Description
+Write Phoenix data through [Jdbc connector](Jdbc.md).
+Support Batch mode and Streaming mode. The tested Phoenix version is 4.xx and 5.xx
+On the underlying implementation, through the jdbc driver of Phoenix, execute the upsert statement to write data to HBase.
+Two ways of connecting Phoenix with Java JDBC. One is to connect to zookeeper through JDBC, and the other is to connect to queryserver through JDBC thin client.
+
+> Tips: By default, the (thin) driver jar is used. If you want to use the (thick) driver  or other versions of Phoenix (thin) driver, you need to recompile the jdbc connector module
+
+> Tips: Not support exactly-once semantics (XA transaction is not yet supported in Greenplum database).
+
+## Options
+
+### driver [string]
+if you use  phoenix (thick) driver the value is  `org.apache.phoenix.jdbc.PhoenixDriver` or you use (thin) driver  the value is `org.apache.phoenix.queryserver.client.Driver`
+
+### url [string]
+if you use  phoenix (thick) driver the value is  `jdbc:phoenix:localhost:2182/hbase` or you use (thin) driver  the value is `jdbc:phoenix:thin:url=http://localhost:8765;serialization=PROTOBUF`
```

Comment:
update `if you use<space><space>phoenix` to `if you use<space>phoenix`

Reply:
> update `if you use<space><space>phoenix` to `if you use<space>phoenix`

thanks，done